tf-modules/cumulus-message-adapter/README.md

```diff
@@ -0,0 +1,20 @@
+# Cumulus Message Adapter
+
+This pacakge creates a lambda layer for upload/use of the [Cumulus Message Adapter](https://github.com/nasa/cumulus-message-adapter).    This can be utilized in combination with the kes deployment tool per the [input/output Cumulus documentation](http://localhost:3000/cumulus/docs/next/workflows/input_output).
+
+## Deployment
+
+1. Copy the .tfvars sample file: `cp terraform.tfvars.sample terraform.tfvars`
+2. Populate the sample file with values that apply to your AWS environment (see configuration variables section below).
+3. Deploy this module: `terraform apply`
+
+NOTE: Terraform will ignore the `aws_profile` config variable if you have static credentials or environment variables set, see the [AWS Provider page](https://www.terraform.io/docs/providers/aws/index.html#authentication).
+
+## Configuration
+
+Configuration variables are shown in `terraform.tfvars.sample`, and are explained below:
+
+```text
+# Required
+cma_version     = "executionsTableName" #  CMA Release version to deploy.
```

Comment:
Seems like the example value here should be updated

Reply:
d46a439a1